include/odp/api/spec/cpu.h

```diff
@@ -143,19 +148,23 @@ uint64_t odp_cpu_cycles_diff(uint64_t c2, uint64_t c1);
 /**
  * Maximum CPU cycle count
  *
- * Maximum CPU cycle count value before it wraps back to zero.
+ * Maximum CPU cycle count value before it wraps back to zero. Returns zero
+ * if CPU cycle counter is not supported.
  *
  * @return Maximum CPU cycle count value
+ * @retval 0 Not supported
  */
 uint64_t odp_cpu_cycles_max(void);
 
 /**
  * Resolution of CPU cycle count
  *
  * CPU cycle count may advance in steps larger than one. This function returns
- * resolution of odp_cpu_cycles() in CPU cycles.
+ * resolution of odp_cpu_cycles() in CPU cycles. Returns zero if CPU cycle
+ * counter is not supported.
  *
  * @return CPU cycle count resolution in CPU cycles
+ * @retval 0 Not supported
  */
 uint64_t odp_cpu_cycles_resolution(void);
```

Comment:
Matching validation test change is missing.

Reply:
Fixed in v3